src/content/blog/podman-and-docker-rootless.md

```diff
@@ -0,0 +1,481 @@
+   ```
+
+5. Verify the socket:
+
+   ```bash
+   ls $XDG_RUNTIME_DIR/docker.sock
+   ```
+
+   You should see `/run/user/1000/docker.sock` (the number may vary).
+
+6. Configure Docker context (if needed):
+
+   ```bash
+   # View existing contexts
+   docker context ls
+
+   # Create rootless context
+   docker context create rootless \
+       --description "Rootless runtime socket" \
+       --docker host="unix://$XDG_RUNTIME_DIR/docker.sock"
+
+   # Switch to the context
+   docker context use rootless
+
+   # Verify it works
+   docker ps
+   ```
+
+7. Docker Rootless requires no-bind-mounts mode
+
+   Docker Rootless has a limitation with bind mounts that affects DDEV. You must enable `no-bind-mounts` mode:
+
+   ```bash
+   ddev config global --no-bind-mounts=true
+   ```
+
+   **Why this is needed**:
+
+   Docker Rootless sets ownership for bind mounts to `root` inside containers. This is a known issue:
+   - [Mounting a volume with rootless always assigns ownership to root](https://github.com/moby/moby/issues/45919)
+   - [Add ability to mount volume as user other than root](https://github.com/moby/moby/issues/2259)
+
+   The `root` user inside the container maps to your host user, but many services will not run as root:
+   - nginx runs as root without problems
+   - MySQL/MariaDB need extra configuration
+   - Apache and PostgreSQL will not run as root
+
+   Podman Rootless fixes this with the `--userns=keep-id` option, which keeps user IDs the same. Docker Rootless does not have this option.
+
+   The `no-bind-mounts` mode fixes this by using Mutagen for the `web` container.
+
+## Running Multiple Container Runtimes
+
+You can run Docker and Podman sockets simultaneously and switch between them using Docker contexts.
+
+For example, here's a system with four active Docker contexts:
+
+```bash
+$ docker context ls
+NAME                DESCRIPTION                               DOCKER ENDPOINT
+default             Current DOCKER_HOST based configuration   unix:///var/run/docker.sock
+podman              Podman (rootful)                          unix:///var/run/podman/podman.sock
+podman-rootless *   Podman (rootless)                         unix:///run/user/1000/podman/podman.sock
+rootless            Rootless runtime socket                   unix:///run/user/1000/docker.sock
+```
+
+Switch between them with:
+
+```bash
+docker context use <context-name>
+```
+
+**Note**: Running both Docker and Podman in rootful mode at the same time may cause network conflicts. See [Podman and Docker network problem on Fedora 41](https://github.com/containers/podman/issues/24486).
+
+## Switching Runtimes with DDEV
+
+DDEV automatically detects your active container runtime. To switch:
+
+1. Stop DDEV projects:
+
+   ```bash
+   ddev poweroff
+   ```
+
+2. Switch Docker context or change the `DOCKER_HOST` environment variable
+
+3. Start your project:
+
+   ```bash
+   ddev start
+   ```
+
+## Which Runtime Should You Choose?
```

Comment:
Great section. However, it should encourage standard Docker setup first, with link to the docs, as it's by far the best for local development.